src/test_driver/mbed-functional/requirements.txt

```diff
@@ -0,0 +1,49 @@
+appdirs==1.4.4
```

Comment:
We will have to add this into the pigweed requirements when upstreaming.

Reply:
Okay